package.json

```diff
@@ -78,8 +78,8 @@
     "test:coverage": "npm run test -- --watchAll=false --coverage",
     "lint": "eslint --quiet .",
     "lint:fix": "eslint . --fix",
-    "prettier": "./node_modules/.bin/prettier --config .prettierrc --check 'src/**/*.js'",
-    "prettier:fix": "./node_modules/.bin/prettier --config .prettierrc --write 'src/**/*.js'",
+    "prettier": "./node_modules/.bin/prettier --config .prettierrc --check src/**/*.js",
```

Comment:
@isaqueha is there any difference?

Reply:
Windows execution breaks with the path as with the single quotation marks.
It cannot find any files with `'src/**/*.js'`
With `src/**/*.js`, it works on Win